tasks/notify.py

```diff
@@ -36,6 +39,131 @@
 CONSECUTIVE_THRESHOLD = 3
 CUMULATIVE_THRESHOLD = 5
 CUMULATIVE_LENGTH = 10
+CI_VISIBILITY_JOB_URL = 'https://app.datadoghq.com/ci/pipeline-executions?query=ci_level%3Ajob%20%40ci.pipeline.name%3ADataDog%2Fdatadog-agent%20%40git.branch%3Amain%20%40ci.job.name%3A{}&agg_m=count'
+
+
+class ExecutionsJobInfo:
+    def __init__(self, job_id: int, failing: bool = True, commit: str = None):
+        self.job_id = job_id
+        self.failing = failing
+        self.commit = commit
```

Comment:
not for another PR: you could use a [dataclass](https://docs.python.org/3.11/library/dataclasses.html) for all these class I believe

Reply:
Thanks for the suggestion 👍

Reply:
I applied this suggestion in this PR as it is a simple one 😁